Jonathan-jonathankern/Homework-4/99-bottles-of-beer/js/app.js

```diff
@@ -1,22 +1,21 @@
 'use strict';
 
-// create li element
-const createListItemElement = document.createElement('li');
-
 // find ul
 const findUnorderedListElement = document.querySelector('ul');
 
-// append li to ul
-const appendListItem = findUnorderedListElement.appendChild(createListItemElement);
-// console.log(appendListItem);
-
 // create li text node
 let createListItemTextNode;
 let quote;
 let beerArray = [];
 
 // if i is greater than 0 and less than 100, subtract 1
-for (let i = 99; i >= 0; i--) {
+for (let i = 99; i > 0; i--) {
+	// create li element
+	const createListItemElement = document.createElement('li');
```

Comment:
Nice fix! One note on naming here: starting a variable with a verb generally suggests we should expect the value to be a function. Simply `listItemElement` or `newListItem` might be clearer here as a variable name that's storing a new element.